x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_creation/components/configuration_step/configuration_step_form.tsx

```diff
@@ -554,9 +573,22 @@ export const ConfigurationStepForm: FC<ConfigurationStepProps> = ({
         <Fragment>
           <EuiFormRow
             fullWidth
-            label={i18n.translate('xpack.ml.dataframe.analytics.create.dependentVariableLabel', {
-              defaultMessage: 'Dependent variable',
-            })}
+            label={
+              <OptionLabelWithIconTip
+                label={i18n.translate(
+                  'xpack.ml.dataframe.analytics.create.dependentVariableLabel',
+                  {
+                    defaultMessage: 'Dependent variable',
+                  }
+                )}
+                tooltip={i18n.translate(
+                  'xpack.ml.dataframe.analytics.dependentVariableInfoTooltip',
+                  {
+                    defaultMessage: 'The field that you want to predict the value of.',
```

Comment:
I wonder if both a tooltip and helptext are overkill? IMO the improved help text is sufficient